<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoteLiteMainPanelClass</class>
 <widget class="QMainWindow" name="NoteLiteMainPanelClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>记事本</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>769</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件(F)</string>
    </property>
    <addaction name="actionNewFile"/>
    <addaction name="actionNewWindow"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionPageSet"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>编辑(E)</string>
    </property>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
    <addaction name="actionFindNext"/>
    <addaction name="actionFindPrev"/>
    <addaction name="actionReplace"/>
    <addaction name="actionGoto"/>
    <addaction name="separator"/>
    <addaction name="actionSelectAll"/>
    <addaction name="actionDateAndTime"/>
   </widget>
   <widget class="QMenu" name="menuFormat">
    <property name="title">
     <string>格式(O)</string>
    </property>
    <addaction name="actionWordWrap"/>
    <addaction name="actionFont"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>查看(V)</string>
    </property>
    <addaction name="actionStatusBar"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助(H)</string>
    </property>
    <addaction name="actionViewHelp"/>
    <addaction name="actionFeedBack"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuFormat"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="statusTip">
    <string>这是一个记事本</string>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionNewFile">
   <property name="text">
    <string>新建(N) Ctrl+N</string>
   </property>
  </action>
  <action name="actionNewWindow">
   <property name="text">
    <string>新窗口(W) Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>打开(O)... Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存(S) Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>另存为(A)... Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionPageSet">
   <property name="text">
    <string>页面设置(U)...</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>打印(P)... Ctrl+P</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出(X)</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>撤销</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>剪切</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>查找</string>
   </property>
  </action>
  <action name="actionFindNext">
   <property name="text">
    <string>查找下一个</string>
   </property>
  </action>
  <action name="actionFindPrev">
   <property name="text">
    <string>查找上一个</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="text">
    <string>替换</string>
   </property>
  </action>
  <action name="actionGoto">
   <property name="text">
    <string>转到</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>全选</string>
   </property>
  </action>
  <action name="actionDateAndTime">
   <property name="text">
    <string>日期和时间</string>
   </property>
  </action>
  <action name="actionWordWrap">
   <property name="text">
    <string>自动换行</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="text">
    <string>字体</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="text">
    <string>缩放</string>
   </property>
  </action>
  <action name="actionStatusBar">
   <property name="text">
    <string>状态栏</string>
   </property>
  </action>
  <action name="actionViewHelp">
   <property name="text">
    <string>查看帮助</string>
   </property>
  </action>
  <action name="actionFeedBack">
   <property name="text">
    <string>发送反馈</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于记事本</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="NoteLiteMainPanel.qrc"/>
 </resources>
 <connections/>
</ui>
